src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write code here
+const inputs = document.querySelectorAll('input');
+const inputClass = 'field-label';
+const firstLatterUp = (word = '') => {
```

Comment:
```suggestion
const firstLetterUp = (word = '') => {
```
Let's get used to name function according to these rules:
![image](https://user-images.githubusercontent.com/96409563/208867018-16bdd139-fc4d-4bcb-b6e4-13f720ef84fb.png)